packages/trader/src/App/Components/Elements/ContractDrawer/contract-drawer.tsx

```diff
@@ -20,12 +22,12 @@ import { observer, useStore } from '@deriv/stores';
 
 type TContractDrawerCardProps = React.ComponentProps<typeof ContractDrawerCard>;
 type TContractDrawerProps = RouteComponentProps & {
+    contract_info?: TContractInfo;
```

Comment:
Seems, that `contract_info` might be optional as on line 93 there was a check for it. That is why I change a type a little bit and add a fallback ({}) + modify check. 
As for card: removed 'no qa attention' tag and add a short checklist for sanity